bootstrap/src/sun/nio/ch/lincheck/EventTracker.kt

```diff
@@ -28,6 +28,7 @@ internal interface EventTracker {
 
     fun beforeReadField(obj: Any, className: String, fieldName: String, codeLocation: Int)
     fun beforeReadFieldStatic(className: String, fieldName: String, codeLocation: Int)
+    fun beforeReadFinalFieldStatic(className: String)
```

Comment:
Since we now tracking both final and non-final fields, maybe instead just leave one method: `beforeReadFieldStatic`  and pass `isFinal: Boolean` as a parameter?

Reply:
We do not analyze accesses to `final` fields, so we don't need to provide `fieldName` and `codeLocation`.